src/main/java/io/openliberty/tools/intellij/util/LibertyGradleUtil.java

```diff
@@ -158,44 +158,82 @@ private static boolean containerVersion(String plugin) {
         return false;
     }
 
-    public static String getGradleSettingsCmd(Project project) {
+    /**
+     * Get Gradle preferences from Build Tools in IntelliJ and return a command to execute or an exception
+     *
+     * @param project liberty project
+     * @return String command to execute in the terminal or an exception to display
+     * @throws IOException
```

Comment:
In many of these methods the comment says it throws IOException but it actually throws LibertyException or none at all. Could you review these comments because it might be confusing later.

Reply:
Good catch!